cryptomatte/cryptomatte.h

```diff
@@ -444,7 +444,7 @@ inline AtString get_user_data(const AtShaderGlobals* sg, const AtNode* node,
     if (pentry) {
         if (AiUserParamGetType(pentry) == AI_TYPE_STRING &&
             AiUserParamGetCategory(pentry) == AI_USERDEF_CONSTANT) {
-            return AiNodeGetStr(node, user_data_name);
+            return AiNodeGetStr(node, AtString(user_data_name));
```

Comment:
I think `user_data_name` is already an AtString.
```suggestion
            return AiNodeGetStr(node, user_data_name);
```